assets/src/dashboard/app/views/editorSettings/adManagement/adNetwork/index.js

```diff
@@ -130,76 +98,44 @@ function AdNetworkSettings({ adNetwork: adNetworkRaw, handleUpdate }) {
     return null;
   }, [adNetwork]);
 
-  const handleMonetizationClick = useCallback(
-    (evt) =>
-      trackClick(evt, 'monetization', 'dashboard', TEXT.HELPER_LINK_NONE),
-    []
-  );
   const handleAdNetworkClick = useCallback(
     (evt) => trackClick(evt, 'monetization', 'dashboard', link),
     [link]
   );
 
   return (
-    <AdNetworkSettingForm onSubmit={(e) => e.preventDefault()}>
-      <div>
-        <SettingHeading>{TEXT.SECTION_HEADING}</SettingHeading>
+    <>
+      <DropDown
+        ariaLabel={TEXT.SLOT_ID_LABEL}
+        options={OPTIONS}
+        selectedValue={adNetwork}
+        onMenuItemClick={(_, newAdNetwork) => handleUpdate(newAdNetwork)}
+        fillWidth={true}
+      />
+
+      {message && (
         <TextInputHelperText
           size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
         >
           <TranslateWithMarkup
             mapping={{
               a: (
                 <InlineLink
-                  href={TEXT.HELPER_LINK_NONE}
+                  href={link}
                   rel="noreferrer"
                   target="_blank"
-                  onClick={handleMonetizationClick}
+                  onClick={handleAdNetworkClick}
                   size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
                   as="a"
```

Comment:
does this need to be `forwardedAs` since it's a styled `Link`?
```suggestion
                  forwardedAs="a"
```

Reply:
good call - these are actually unnecessary altogether now. removed.